R/tm_g_swimlane.R

```diff
@@ -486,8 +485,7 @@ srv_g_swimlane <- function(id,
             }),
             marker_color_opt = .(if (length(marker_color_var) == 0) {
               NULL
-            } else if (length(marker_color_var) > 0 &
-              all(unique(anl[[marker_color_var]]) %in% names(marker_color_opt)) == TRUE) { # nolint
+            } else if (length(marker_color_var) > 0 & all(unique(anl[[marker_color_var]]) %in% names(marker_color_opt))) { # nolint: line_length.
```

Comment:
Alternative to consider

```suggestion
            } else if (
              length(marker_color_var) > 0 &
                all(unique(anl[[marker_color_var]]) %in% names(marker_color_opt))
            ) {
```